### ISSUE TITLE
Legiones Astartes Bugs

### ISSUE THREAD

**Issue by Tomakokamikaze** [BSData/horus-heresy#1643]
Assuault Marines:
-Heavy chainsword (weapon Profile not being shown)

Apothecaries: when choosing SM Bike
-> Toughness +1 not applied
-> Weapon on bike is Bolter (should be Twin Linked Bolter)
-> Not saying the guy is very bulky now

Centurions: Not showing twin linked weapon profile

Outriders & Centurion: Don't have the SM Bike box that the Apothecaries have (either add it to them, or remove from apothecaries)

**Comment by Mayegelt** [comment]
Thanks for the bug report. Just did a bit of work and sorted it out, along with a few other related issues I discovered.

Fixed Heavy Chainsword on Assault Marine Sergeant.

Fixed Twin-Linked Bolters on bikes, and a bunch of other Twin-Linked weapons fixed in GST and LA file, as many missed out on the "Twin-Linked" rule.

Fixed a whole bunch of Bike and Jetbikes missing appropriate rules, such as Jink, Hammer of Wrath, Relentless, Very Bulky.

Fixed Apothecary not getting +1T for Bikes, Also not getting 2+ saves for Artificer Armour
